src/DatePicker/CalendarMonth.js

```diff
@@ -19,12 +21,24 @@ class CalendarMonth extends React.Component {
     return this.selectedDateDisabled;
   }
 
+  handleTouchTapDay = (event, date) => {
+    if (this.props.onTouchTapDay) this.props.onTouchTapDay(event, date);
+  };
+
+  shouldDisableDate(day) {
+    if (day === null) return false;
+    let disabled = !isBetweenDates(day, this.props.minDate, this.props.maxDate);
+    if (!disabled && this.props.shouldDisableDate) disabled = this.props.shouldDisableDate(day);
+
+    return disabled;
+  }
+
   getWeekElements() {
     const weekArray = getWeekArray(this.props.displayDate, this.props.firstDayOfWeek);
 
     return weekArray.map((week, i) => {
       return (
-        <ClearFix key={i}>
+        <ClearFix key={i} style={this.styles.week}>
```

Comment:
Can we get rid of `ClearFix` as a container element? Must be possible without if we are using flexbox now.